go/chat/utils/utils.go

```diff
@@ -754,6 +754,10 @@ func PresentThreadView(ctx context.Context, uid gregor1.UID, tv chat1.ThreadView
 	return res
 }
 
+func computeOutboxOrdinal(obr chat1.OutboxRecord) float64 {
+	return float64(obr.Msg.ClientHeader.OutboxInfo.Prev) + float64(obr.Ordinal)/1000.0
```

Comment:
I'm wary of using floating-point numbers for ordinals, esp. since ints mapped to javascript turn into floats. Are we assuming that we'll never have more than 2^32 messages in a convo?

Reply:
Only pending messages get the fractional ordinal, so you would have to 1000 pending messages after 1 real message to get in trouble here.

Reply:
But it's a good point about maximum message ID, currently that is the limit.